Updated Radstation

Arbitrated opened the pull request:
# Description
Created a functional mail system and updated disposals. Also added APCs to dorms and perma, and removed the unstable teleporting locker from the Maints Bar.

The Infrastructure War has begun.

# Media

The map renderer isn't working for some reason, which is really annoying. It gave us these errors:
https://gist.github.com/Arbitrated/62eb09101660822b319d50296e4e00fd

We ran Unit and Integration tests and they passed (Rain/Ashkitten can confirm, they know a lot more about tech than we do)

EDIT: Dunrab got it to work! 
<img width="5248" height="6112" alt="Rad-0" src="https://github.com/user-attachments/assets/9c93ef44-8ea1-4ac5-bfc5-18d574e1443c" />


---

# Changes
- Completely remade Radstation's disposals system
- Adjusted some doors in Bar and LO's office to make room for mailing units
- Added a functional mail system on Radstation
- Added APCs to dorms and Perma in Radstation
- Added Renault spawner to Captain's room
- Removed the unstable teleporting locker from Radstation's maints bar


# Changelog

:cl:
- tweak: Radstation has been updated. The changes include the addition of a fully functional mailing and disposal systems. See PR#1521 for more info.

Dunrab committed:
dont paint deleted entities (thanks Mnemo for helping with this one)

Dunrab commented:
For your map render failing error, its trying to render a deleted entity, my PR for submarine has a fix for this issue but sadly isn't merged yet. You could make the change from that PR locally and it should render with out an issue.

Arbitrated committed:
Updated Radstation

Created a functional mail system and updated disposals. Also added APCs to dorms and perma, and removed the unstable teleporting locker from the Maints Bar.

Arbitrated commented:
> For your map render failing error, its trying to render a deleted entity, my PR for submarine has a fix for this issue but sadly isn't merged yet. You could make the change from that PR locally and it should render with out an issue.

We tried that, and it doesn't appear to be working still, threw us some more errors: https://gist.github.com/Arbitrated/7ffcc620047bd444cc29b476ffbb7e34

Dunrab commented:
Weird ill try to debug it when I get the chance to see if i can figure out why this is happening.

Arbitrated commented:
UPDATE: Dunrab got the map renderer to work, so now the PR has an image!

Mnemotechnician commented:
Updated your changelog to be more concise and match our standards. The old list of changes will remain above it.

Don't see any issues on the render. I personally think the suspicious closet was a nice addition, but it indeed does need to go because some people abuse it way too much and disrupt the experience for other players while doing so.

Dunrab reviewed (approved):
Changes look good, I tested your new disposals and mail units the few I tested worked just fine. Also sad to see the locker in the maints bar go but I know very well why it has to.

Arbitrated commented:
> Updated your changelog to be more concise and match our standards. The old list of changes will remain above it.
> 
> Don't see any issues on the render. I personally think the suspicious closet was a nice addition, but it indeed does need to go because some people abuse it way too much and disrupt the experience for other players while doing so.

Thanks Mnemno! We didn't catch that there was a special format here for changelogs, we'll keep that in mind for any future PRs. 

Do we need to be concerned about the failed build GitHub is giving us?
